conf/ocp_version/ocp-4.7-config.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# Config file for nightly OCP 4.7
+RUN:
+  client_version: "4.7.0-0.nightly"
+DEPLOYMENT:
+  installer_version: "4.7.0-0.nightly"
+  terraform_version: "0.12.26"
+ENV_DATA:
+  vm_template: "rhcos-46.82.202010011740-0-vmware.x86_64"
```

Comment:
What about to add some todo here about change to 47 rhcos image?

Reply:
I guess it will be changed in a while to some image which will contain 47 there, so we need to keep eyes on this, so maybe worth to open issue or write some note here. Maybe @vavuthu remember when they are changing to the proper version?

Reply:
I am not sure the timelines when they changes the RHCOS  version. I will raise issue once this PR is merged.